### ISSUE TITLE
<CAIA EPIC TEST> from <CAIA>

### ISSUE THREAD

**Issue by coforma-terry** [department-of-veterans-affairs/va.gov-team#62609]
## About your team

- Team name: 
-	OCTO-DE product owner: 
-	Product manager: 
-	Designer: 
- Accessibility specialist:
- Content writer: 
-	Slack channel:
 
### Describe what you need in a few sentences:

</br>
</br>
</br>

#### Examples
- We’re updating a form and want to confirm we’re doing it correctly.
- We’re launching a new online tool to VA.gov and need support with our user flows.
- We need to make sure the content we’re using in our tool is written in plain language and follows the style guide.

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [ ] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] Object-Oriented UX (OOUX) support
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [ ] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next step is** ....
- **Provide date if available**: xx/xx/xxxx

#### Timing for your next step: 
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [ ] No

### If you're conducting research, when is that starting? 
**Provide date if available:** xx/xx/xxxx
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [ ] Yes
  - [ ] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users on:** xx/xx/xxxx
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [ ] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- **Ticket (number)**: 
- **Collab. Cycle Ticket Opened (date)**: xx/xx/xxxx
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Product outline**:
- **User flows**:
- **Prototype or mockups**:
- **If this is in Sketch, which version or specific page should we focus on?**
- **Content source of truth**:
- **Research plan:**
- **Any epics/issues that may be helpful**:
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [ ] Let’s work async and meet as needed.

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.
